backend/contact/database/datatypes.go

```diff
@@ -1,28 +1,28 @@
 package database
 
 import (
-	"database/sql/driver"
-	"encoding/json"
-  "fmt"
+// 	"database/sql/driver"
+// 	"encoding/json"
   "time"
 
 	"github.com/google/uuid"
 	"gorm.io/gorm"
 )
-
-type JSONB map[string]interface{}
-
-func (j JSONB) Value() (driver.Value, error) {
-	valueString, err := json.Marshal(j)
-	return string(valueString), err
-}
-
-func (j *JSONB) Scan(value interface{}) error {
-	if err := json.Unmarshal([]byte(fmt.Sprintf("%v", value)), &j); err != nil {
-		return err
-	}
-	return nil
-}
+//
```

Comment:
you could just delete these. There is a low chance you will ever need it. Even if you do, you still have the git history :)